docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
+----------------
+
+.. _collection-list:
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+This endpoint allows you to list all collections authored by this user.
+Access is limited to your own collections, or users with `Users:Edit` permission.
```

Comment:
No warning, but they're rendered as _italic_ rather than `code` .  The rest of docs, from a quick look, is inconsistent on how we talk about permissions.  Some of them don't italicise the permission (Reviews) and others don't explicitly mention the string, only that it requires reviewer permissions (Add-ons).